docs/source/yelpsoa_configs.rst

```diff
@@ -400,81 +417,107 @@ Each job configuration MAY specify the following options:
+  * ``deploy_group``: Same as ``deploy_group`` for marathon-*.yaml.
 
-  * ``extra_volumes``: See the `marathon-[clustername].yaml`_ section for details
+  * ``schedule_time_zone``: The time zone name to use when scheduling the job.
+    Unlike schedule, this is specified in the tz database format, not the ISO 8601 format.
+
+    * This field takes precedence over any time zone specified in schedule.
+    * See list of `tz database time zones <https://en.wikipedia.org/wiki/List_of_tz_database_time_zones>`_.
+    * For example, the effective time zone for the following is America/Los_Angeles::
 
-  * ``constraints``: Array of rules to ensure jobs run on slaves with specific
-    Mesos attributes. See the `official documentation
-    <https://mesos.github.io/chronos/docs/api.html#constraints>`_ for more
-    information.
 
-  * ``extra_constraints``: Adds to the default placement constraints for
-    services. This acts the same as ``constraints``, but adds to the default
-    constraints instead of replacing them. See ``constraints`` for details on
-    format and the default constraints.
+        ---
+        main:
+          schedule: R/2014-10-10T18:32:00Z/PT60M
+          schedule_time_zone: America/Los_Angeles
 
-    *Note*: While this parameter is the same as ``extra_constraints`` in ``marathon-$cluster.yaml``,
-    the Marathon constrain language isn't exactly like the Marathon constraint language.
-    Be sure to read the constraint documentation for Chronos referenced in the ``constraints``
-    section.
+``tron-[tron-clustername].yaml``
+------------------------------
 
-  * ``pool``: See the `marathon-[clustername].yaml`_ section for details
+This file stores periodically scheduled jobs for execution on `Tron <https://github.com/yelp/tron>`_.
 
-  * ``deploy_whitelist``: See the `marathon-[clustername].yaml`_ section for details
+The documentation here is for the PaaSTA-specific options. For all other
+settings, please see the
+`canonical docs <https://tron.readthedocs.io/en/latest/jobs.html>`_.
 
-  * ``deploy_blacklist``: A list of lists indicating a set of locations to *not* deploy to. For example:
+.. warning:: The PaaSTA-Tron Integration is currently in an ALPHA state. Do not use it unless directed to.
 
-      ``deploy_blacklist: [["region", "uswest1-prod"]]``
+Example Job
+^^^^^^^^^^^
 
-   would indicate that PaaSTA should not deploy the service to the ``uswest1-prod`` region. By default the ``monitoring_blacklist`` will use the ``deploy_blacklist`` if it exists.
+::
 
+  jobs:
+      - name: convert_logs
+        node: node1
+        schedule:
+          start_time: 04:00:00
+        actions:
+          - name: verify_logs_present
+            command: "ls /var/log/app/log_%(shortdate-1).txt"
+             executor: paasta
+          - name: convert_logs
+            command: "convert_logs /var/log/app/log_%(shortdate-1).txt /var/log/app_converted/log_%(shortdate-1).txt"
```

Comment:
`%(shortdate-1)s` (also in the above command)

Reply:
Whoa uh, I copy pasted this from our own (commented out) configs... I'll fix them there too!